bridgetown-routes/lib/bridgetown-routes/code_blocks.rb

```diff
@@ -30,7 +30,7 @@ def eval_route_file(file, file_slug, app) # rubocop:disable Lint/UnusedMethodArg
 
           code = File.read(file)
           code_postmatch = nil
-          ruby_content = code.match(Bridgetown::FrontMatterImporter::RUBY_BLOCK)
+          ruby_content = code.match(Bridgetown::FrontMatter::Loaders::Ruby::BLOCK)
```

Comment:
**comment:** This appears to be dealing with front matter without really stating that fact. Perhaps we should refactor it?